### DISCUSSION TITLE
Training the model

### DISCUSSION THREAD

**Discussion by morpheuslord** [microsoft/BitNet#45] [Q&A]
Hello there, I am extremely new  to LLM training and training large models in general, can you give me general directions on how I can do that and how I can train this model or fine-tune it to work for my specific use case.

**Comment by AiSatoshiBasilisk** [comment]
https://huggingface.co/blog/1_58_llm_extreme_quantization

**Comment by azmatsiddique** [comment]
Begin with a strong open-source base model (e.g., LLaMA/Mistral-class) and fine-tune it using LORA/QLORA on a small, high-quality dataset tailored to your task
RAG with prompt engineering beats fine-tuning and is faster to iterate.

Best Course:
https://www.udemy.com/course/llm-engineering-master-ai-and-large-language-models
